README.md

```diff
@@ -55,7 +55,7 @@ class UserController extends Controller
 }
 ```
 
-You can also check our `DemoController`
+You can also check our [DemoController](https://github.com/kjonski/HashId/blob/dev-master/src/Controller/DemoController.php).

```

Comment:
Good point but it could be better to use relative path [DemoController](src/Controller/DemoController.php)

Reply:
Changed.